lib/calendars.js

```diff
@@ -9,6 +9,15 @@ import * as webdav from './webdav';
 
 let debug = require('./debug')('dav:calendars');
 
+const ICAL_OBJS = new Set([
```

Comment:
Accidental modification, primarily. I figured that since it's a constant, it'd might as well be a `const`. Immutability improves safety and the ability to reason about the code. If it needs to change (unless supporting vendor extensions, it won't), it can just be made to use `let`.